server/routes/api.js

```diff
@@ -59,8 +59,8 @@ router.post('/api/*', async ctx => {
   const chooseToken = usingNoAuthToken ? unAuthToken : authToken;
 
   // check if token expired, retrieve refresh token or special token
-  const { expires_in } = chooseToken;
```

Comment:
No need to change. Why you delete `expires_in` check ?

Reply:
If cookie is expired, expires_in is undefined, compared with now time is no effect. Correct judge  condition is refresh_token has value and not get access_token.

Reply:
Cookie expired time set value  according to expires_in, check expires time by whether get 'access_token' or other cookies.